contracts/src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol

```diff
@@ -665,14 +665,14 @@ contract NOPsSettlement is SetUp {
     registry.transmit(reportContext, reportBytes, rs, ss, vs);
 
     // verify transmitters have positive balances
-    (bool active, uint8 index, uint96 balance, uint96 lastCollected, ) = registry.getTransmitterInfo(TRANSMITTERS[1]);
-    assertTrue(active);
-    assertEq(1, index);
-    assertTrue(balance > 0);
-    assertEq(0, lastCollected);
-
     uint256[] memory payments = new uint256[](TRANSMITTERS.length);
-    for (uint256 i = 0; i < payments.length; i++) {
+    for (uint256 i = 0; i < TRANSMITTERS.length; i++) {
+      (bool active, uint8 index, uint96 balance, uint96 lastCollected, ) = registry.getTransmitterInfo(TRANSMITTERS[i]);
```

Comment:
had to change this because not every transmitter earns the same amount (the one sending the tx earns more because of gas cost compensation)